### ISSUE TITLE
Empty historical parameter data export when using only UDP Parameter Data Links

### ISSUE THREAD

**Issue by acommend** [yamcs/yamcs#984]
When using version 5.10.7 configured to only use UDP Parameter Data Links, the Web interface export for historical parameter data produces a file with no data in it. We started with the example in the "Getting Started" documentation but modified the instance config file to only enable a UDP Parameter Data Link.

Screen shot of historical parameter data:

![image](https://github.com/user-attachments/assets/6b96f9a1-a348-4029-85df-1d329b56ec94)

Screen shot of instance data link configuration:

![image](https://github.com/user-attachments/assets/e6f06d9d-12b7-4bcf-8db6-ee8877944898)

**Comment by xpromache** [comment]
Yeah Lorenzo is right that the export is doing a replay and I'm revamping right now the parameter retrieval to be more consistent among those calls (in this case it would return from the parameter archive instead of doing a replay).

Still the replay should return the data. Please check in the PP table that you have data (yamcs-web->Admin Area-> Databases->Tables).

I suspect something with the groups. Try putting "recordingGroup: /myproject/" as argument to your link (next to "port:" and "json:") 

These recording groups is something that also requires some revamping in Yamcs: the replays assume that it can filter on the pp tables based on the recording group but at the injection time it is not enforced that a parameter uses its own group (which is not even possible to define in the XTCE so it takes the subystem name as the group).
